lib/wpscan.rb

```diff
@@ -38,7 +38,15 @@ module WPScan
   include CMSScanner
 
   APP_DIR = Pathname.new(__FILE__).dirname.join('..', 'app').expand_path
-  DB_DIR  = Pathname.new(Dir.home).join('.wpscan', 'db')
+
+  # XDG support for DB_DIR
+  legacy_path = Pathname.new(Dir.home).join('.wpscan', 'db')
+  xdg_path = Pathname.new(ENV['XDG_CACHE_HOME'] || Dir.home.join('.cache')).join('wpscan', 'db')
```

Comment:
`Dir.home.join(...)` does not work for me. Perhaps this should be `Pathname.new(Dir.home).join(...)` instead?

Reply:
Good catch. Fixed